### DISCUSSION TITLE
GitHub commits not showing on netlify website

### DISCUSSION THREAD

**Discussion by AndersBear** [community/community#47201] [New to GitHub]
Ever since a couple months ago, my commits aren't going through to my webpage.  

GitHub editor shows some pink and blue areas with plus and minus signs.  I don't know what that means.  (The format changed.  I'm using a free account.  Maybe it expired?)

GitHub also shows links like "Update xxxxx.html."  When I click and edit, the commits still don't go through.  The netlify webpage still shows the old source when I right-click and "view page source."  I started a new repository and a new webpage and they work just fine, including edits.

?

**Comment by AndersBear** [comment]
On my netlify page, there was a little exclamation point next to this particular project.  I needed to select a new build image.  I think that fixed it.

**Comment by NJacksonDev** [comment]
Below are some reasons why a commit does not go through to a netlify website (one of which is related to the build process, which it seems you fixed). Could be useful for the future in case you have any additional issues.

- The site is not configured to build automatically: Netlify offers an automatic build feature that updates your site whenever a new commit is made to your GitHub repository. If this feature is not enabled, you will have to manually trigger the build process to see the changes on your site.

- The build process failed: If there was an error during the build process, Netlify will not update your site with the latest changes. You can check the build log to see if there were any errors or warnings during the build process.

- The deployment branch is incorrect: By default, Netlify deploys the master branch of your GitHub repository. If you made the commit to a different branch, you will need to configure Netlify to deploy that branch instead.
